### PR TITLE
add bgcolor to turtul module

### PR DESCRIPTION
**Pull request by cartoone222**

### CODE REVIEW

**Comment by cartoone222** [comment]
is for do this issue https://github.com/UpsilonNumworks/Upsilon/issues/301

File: python/port/mod/turtle/turtle.h [hunk 1]
```diff
@@ -99,6 +108,7 @@ class Turtle {
   static constexpr uint8_t k_defaultSpeed = 8;
   static constexpr uint8_t k_maxSpeed = 10;
   static constexpr KDColor k_defaultColor = KDColorBlack;
+  static constexpr KDColor k_defaultColorBG = KDColorYellow;
```
Comment (Yaya-Cout) [inline python/port/mod/turtle/turtle.h]: Why are you overwriting the default color ? You should rather use the theme value.

**Review by Yaya-Cout — CHANGES_REQUESTED** [review]
Also, can you add the function to the toolbox ? It should also automatically setup the auto-completion, if I remember correctly. If you don't know how to do that, it's not a problem, it's not required for it to work and I can do it myself.

File: python/port/mod/turtle/turtle.h (hunk 1 above)
Comment (cartoone222) [reply python/port/mod/turtle/turtle.h]: sory is an forgoten element of debuginig. also yes i can try to add to the toolbox i try to take time to try on thats in the next day

File: python/port/mod/turtle/turtle.h [hunk 2]
```diff
@@ -99,6 +108,7 @@ class Turtle {
   static constexpr uint8_t k_defaultSpeed = 8;
   static constexpr uint8_t k_maxSpeed = 10;
   static constexpr KDColor k_defaultColor = KDColorBlack;
+  static constexpr KDColor k_defaultColorBG = KDColorWhite;
```
Comment (Yaya-Cout) [inline python/port/mod/turtle/turtle.h]: Default BG is not always white, it depend of the theme. For example, using Omega Kawaii (left is current Upsilon version and right is your version) : 
Left : Normal simulator, with Omega Kawaii Background ; Right : Simulator built from this pull request, with white background
You can add `THEME_NAME=omega_kawaii THEME_REPO=https://github.com/Omega-Numworks-Prod/Omega-Kawaii-Theme` to your build command to test, after doing a full clean (`make cleanall`).
You should check, but I think the default background color is `Palette::CodeBackground`.

**Review by Yaya-Cout — CHANGES_REQUESTED** [review]
The toolbox seem right. You only need to change the default background color.

**Review by Yaya-Cout — CHANGES_REQUESTED** [review]
Everything is OK, but you should just remove the Omega Kawaii Theme from the repo (themes should only be cloned at build time)

**Review by Yaya-Cout — APPROVED** [review]
LGTM, will merge after the CI.

**Pull request merged**